.github/workflows/inspektor-gadget.yml

```diff
@@ -697,6 +697,48 @@ jobs:
           rm -rf /tmp/.buildx-cache
           mv /tmp/.buildx-cache-new /tmp/.buildx-cache
 
+  analyze-and-build:
+    name: IG binary size analysis
+    runs-on: ubuntu-latest
+    needs: 
+      - build-ig 
+    steps:
+      - name: Set up Go
+        uses: actions/setup-go@3041bf56c941b39c61721a86cd11f3bb1338122a # v5.2.0
+        with:
+          go-version: '1.23.4'
+          cache: true
+        id: go
+      - name: Install go-size-analyzer
+        run: |
+          go install github.com/Zxilly/go-size-analyzer/cmd/gsa@e47aa2561e70b937f2be7441b2eeca984ff60e8b
```

Comment:
nit: please add a comment indicating which version it points to as done in #3199